bin/opcodes/utils.mts

```diff
@@ -1,7 +1,7 @@
-import { existsSync, readFileSync, writeFileSync } from 'node:fs';
-import { dirname, relative, resolve } from 'node:path';
 import chalk from 'chalk';
 import { spawnSync } from 'node:child_process';
+import { existsSync, readFileSync, writeFileSync } from 'node:fs';
+import { dirname, relative, resolve } from 'node:path';
```

Comment:
@NullVoxPopuli this

Reply:
https://github.com/glimmerjs/glimmer-vm/pull/1507

Reply:
this is just a mistake, and there is nothing I can do about it other than manual rebasing -- went over the lints with yehuda, and things look correct.